examples/pytorch/graphsage/link_pred.py

```diff
@@ -11,6 +12,11 @@
 # (This is a long-standing issue)
 from ogb.linkproppred import DglLinkPropPredDataset
 
+parser = argparse.ArgumentParser()
+parser.add_argument('--pure-gpu', action='store_true',
+                    help='mix CPU and GPU training')
```

Comment:
```suggestion
                    help='Perform both sampling and training on GPU.')
```

Reply:
Something went wrong with the copy-pastes, fixed!